### PR TITLE
Ecnconfig

### PR DESCRIPTION
**Pull request by wendani**
**- What I did**
Changed ECN config in qos.json
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: device/arista/x86_64-arista_7050_qx32s/Arista-7050-QX-32S/qos.json [hunk 1]
```diff
@@ -115,24 +115,23 @@
...
+            "wred_red_enable":"true",
             "ecn":"ecn_all",
-            "red_max_threshold":"512000",
-            "red_min_threshold":"512000",
-            "yellow_max_threshold":"512000",
-            "yellow_min_threshold":"512000",
-            "green_max_threshold": "184320",
-            "green_min_threshold": "184320"
+            "red_max_threshold":"312000",
+            "red_min_threshold":"104000",
```
Comment (lguohan) [inline device/arista/x86_64-arista_7050_qx32s/Arista-7050-QX-32S/qos.json]: are we sure we want to different value for min and max?

**Comment by lguohan** [comment]
can you also change for Force10-S6000 platform?

File: device/arista/x86_64-arista_7050_qx32s/Arista-7050-QX-32S/qos.json (hunk 1 above)
Comment (wendani) [reply device/arista/x86_64-arista_7050_qx32s/Arista-7050-QX-32S/qos.json]: This is based on the test report recommendation from other team.

**Comment by wendani** [comment]
Yes. Once confirmed correct. We can also apply it Arista-7050-QX32.

**Pull request merged**